<launch>
    <arg name="mav_name" default="firefly"/>

    <!--- Trajectory generator -->
    <node name="trajectory_generator" pkg="dla4_final_demo" type="trajectory_generator_ros_node" output="screen">
        <!-- <rosparam command="load" file="$(find mav_trajectory_generation_example)/cfg/$(arg mav_name)_params.yaml"/> -->
        <!-- <remap from="uav_pose" to="odometry_sensor1/odometry"/> -->
        <param name="max_v" value="2.0"/>
        <param name="max_a" value="0.5"/>
        <param name="max_j" value="0.75"/>
        <param name="max_s" value="1.5"/>
    </node>

    <!--- Trajectory sampler -->
    <node name="sampler" pkg="dla4_final_demo" type="trajectory_sampler_ros_node" output="screen">
        <remap from="trajectory_4D" to="/trajectory_generator/smooth_trajectory"/>
    </node>
    

    <!--
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find mav_trajectory_generation_example)/cfg/rviz_view_$(arg mav_name).rviz" />
    -->
</launch>